### PR TITLE
[CUBRIDQA-58] add sql_by_cci into ctp

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 

I add the first version of sql_by_cci into ctp

### CODE REVIEW

File: CTP/sql/bin/run.sh [hunk 1]
```diff
@@ -167,6 +169,10 @@ function do_init()
 	 is_support_ha="yes"
     fi
 
+    if [ "x${sql_interface_type}" != "x" ];then
```
Comment (fanzaiqiang) [inline CTP/sql/bin/run.sh]: It's better to remove "x" from if stmt.

File: CTP/sql/bin/run.sh [hunk 2]
```diff
@@ -312,20 +325,48 @@ function config_cubrid_without_ha()
...
 {
      curDir=`pwd`
-     build_ver_type=""
-     qa_db_xml_path=${CTP_HOME}/sql/configuration/Function_Db/${db_name}_qa.xml
-     avaliable_broker_port=`awk '/SERVICE[[:space:]]*=[[:space:]]*ON/, /BROKER_PORT/' $CUBRID/conf/cubrid_broker.conf|grep BROKER_PORT|grep -v '#'|awk -F '=' '{print $2}'|head -1| tr -d ' '`
-     db_url="<dburl>jdbc:cubrid:localhost:${avaliable_broker_port}:${db_name}:::</dburl>"
-     sed -i "s#<dburl>.*</dburl>#$db_url#g" $qa_db_xml_path
-     if [ "$cubrid_bits" == "32" ];then
-          build_ver_type="32bits"
+     if [ "$interface_type" == "cci" ];then
```
Comment (fanzaiqiang) [inline CTP/sql/bin/run.sh]: 1. Move these codes to Makefile.

2. How to check whether support cci_set_cas_change_mode function:
cat $CUBRID/include/cas_cci.h | grep "cci_set_cas_change_mode"  | wc -l

**Comment by supler** [comment]
@fanzaiqiang 

generate summary_info file and refine makefile

**Comment by supler** [comment]
@fanzaiqiang 

change compile as bash shell

**Comment by fanzaiqiang** [comment]
@supler 

Great. Please merge all codes.

**Pull request merged**